packages/transition-frontend/src/components/forms/preferences/sections/PreferencesSectionFeatures.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2023, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import React from 'react';
+import Collapsible from 'react-collapsible';
+import { withTranslation } from 'react-i18next';
+import _toString from 'lodash.tostring';
+import PreferencesResetToDefaultButton from '../PreferencesResetToDefaultButton';
+import InputWrapper from 'chaire-lib-frontend/lib/components/input/InputWrapper';
+import { InputCheckboxBoolean } from 'chaire-lib-frontend/lib/components/input/InputCheckbox';
+import PreferencesSectionProps from '../PreferencesSectionProps';
+
+const PreferencesSectionFeatures: React.FunctionComponent<PreferencesSectionProps> = (
+    props: PreferencesSectionProps
+) => {
+    const prefs = props.preferences.getAttributes();
+
+    return (
+        <Collapsible trigger={props.t('main:preferences:ExperimentalFeatures')} open={true} transitionTime={100}>
+            <div className="tr__form-section">
+                <InputWrapper twoColumns={true} label={props.t('main:preferences:MapPrettyDisplay')}>
+                    <InputCheckboxBoolean
+                        id={'formFieldPreferencesFeatureMapPrettyDisplay'}
```

Comment:
Eventually, this list should be auto generated from a feature list defined elsewhere. (So that we don't have to change the UI for each experimental feature we add)